php/actions/get_filtered_list_topics.php

```diff
@@ -834,11 +839,69 @@
     ]);
 }
 
-function get_client_name( int|null $clientID, array $cfg): string
+/**
+ * Собрать имя клиента
+ *
+ * @param      int|null  $clientID  The client id
+ * @param      array     $cfg       The configuration
+ *
+ * @return     string    The client name.
+ */
+function get_client_name(int|null $clientID, array $cfg): string
 {
     if (!$clientID || !isset($cfg['clients'][$clientID])) return '';
     return sprintf(
         '<i class="client bold text-success">%s</i>',
         $cfg['clients'][$clientID]['cm']
     );
 }
+
+/**
+ * Собрать заголовок для раздачи, в зависимости от её состояния
+ *
+ * @param      string  $bulletState  Статус раздачи
+ * @param      string  $bulletColor  Цвет раздачи (средние сиды)
+ *
+ * @return     string  Заголовок раздачи
+ */
+function get_topic_title(string $bulletState, string $bulletColor = ""): string
+{
+    $topicsBullets = [
+        "fa-arrow-up"   => "Раздаётся",
+        "fa-arrow-down" => "Скачивается",
+        "fa-pause"      => "Приостановлена",
+        "fa-circle"     => "Нет в клиенте",
+        "fa-times"      => "C ошибкой в клиенте"
+    ];
+    $topicsColors = [
+        "text-success" => "полные данные о средних сидах",
+        "text-warning" => "неполные данные о средних сидах",
+        "text-danger"  => "отсутствуют данные о средних сидах"
```

Comment:
Maybe this variant will be more clear to user about what's going on under the hood and how this is calculated?
```suggestion
        "text-success" => "данные о средних сидах накоплены за весь выбранный период",
        "text-warning" => "данные о средних сидах накоплены более чем за половину выбранного периода",
        "text-danger"  => "данные о средних сидах не накоплены даже за половину выбранного  периода"
```

Reply:
IMO, a lot of superfluous information